### ISSUE TITLE
Improve the visual design of the signup screen

### ISSUE THREAD

**Issue by bryankennedy** [cns-iu/xmacroscope#176]
- Center form on screen
- Implement some aspects of the Sportsology design scheme, matching the MAV screen

**Comment by bryankennedy** [comment]
I removed this from this milestone because Bruce made some changes directly on his branch to match up with the MAV. I'll revisit this at a later date.

**Comment by bherr2** [comment]
PS, it looks like there is a Material UI library for React that would match the Material UI components we're using in Angular. Its on v3.0.x... https://material-ui.com/

**Comment by azuldev1** [comment]
@bryankennedy  and @bherr2 I'm not sure if we're closing issues via git reserved words in commits. But I marked this issue as "Fixed" which would then auto close the issue once it is merged into the default branch. I'm not sure if we have the default branch set to develop or master.

[Closing Issues via Commit Messages](https://blog.github.com/2013-01-22-closing-issues-via-commit-messages/)

**Comment by bherr2** [comment]
I like the idea of commit messages closing issues, but realistically we need to have a review/qa process on issues before they're "done done". But definitely reference issues in commit messages.

**Comment by bryankennedy** [comment]
Is the auto-close on "Fixed" keyword set for the develop or master branch? I can't remember if that's a setting you can change via GitHub or it's always only on master.

**Comment by bherr2** [comment]
It must be master because I haven't changed it (nor do I know where its at)

**Comment by bherr2** [comment]
If its not obvious, I assume Bryan will do the Review/QA for SMM work.

**Comment by bryankennedy** [comment]
Manuel and I just talked about this one. There's a few design elements from the original SMM Run exhibit graphics that we're going to build into the start line screen as well. He's working on implementing those now.

**Comment by bherr2** [comment]
For now, we can wild-west the UI design, but we _have_ to come together and do a real design phase for this. Lisel and I are going to propose another goal for Phase 2, which is to do a proper design on the whole UI (including start line screen). The deliverable being a design document (no coding) which we will use to refactor the UI. This will mainly be handled by Leonard, in concert with your design folks, Katy, Lisel, and others.

**Comment by bryankennedy** [comment]
That sounds good to me. As part of that I actually think we should discuss making an "unstyled" version of the design that is focused on functionality, and then think about a theme layer for the colors, animations, etc that we might overlay for each individual install. Baking the sportsology orange, black, and font choices into the xMacroscope might not be the best design choice.

We're going to run into a bit of a technical issue here as well. I believe that you're pretty invested in Google's Material Design and the UI choices that come along with that (animations, button shapes, etc.). This is different than the SMM approach with our design work on this and other projects. Let's flag this for discussion when we focus on the design approach in Phase 2 or 3 (or both).

**Comment by bherr2** [comment]
Yeah, some amount of themeing should be a requirement when designing the new UI. We're definitely all in on Material Design, but yeah we can talk about how to reconcile this with the start line/registration screen.

**Comment by bryankennedy** [comment]
Our phase 1 work for this ticket is complete. We'll do some more updates to the visual design in the next phase of work, under specific tickets.
